Dockerfile.erb

```diff
@@ -34,12 +34,13 @@ RUN ./install-stunnel.sh
 
 # Update http output gem
 # Install our syslog output implementation
-RUN apk add --no-cache git && \
+RUN apk add --no-cache git sed && \
     GEMFILE="/logstash-${LOGSTASH_VERSION}/Gemfile" && \
     grep -v 'logstash-output-http' "$GEMFILE" > "${GEMFILE}.tmp" && \
     mv "${GEMFILE}.tmp" "$GEMFILE" && \
     grep -v 'logstash-output-redis' "$GEMFILE" > "${GEMFILE}.tmp" && \
     mv "${GEMFILE}.tmp" "$GEMFILE" && \
+    sed -i 's#https:\/\/rubygems#http:\/\/rubygems#g' $GEMFILE && \
```

Comment:
This is pretty unacceptable, and I'm still working on a fix. See the slack convo here for what has been tried: https://aptible.slack.com/archives/C02NMLZ8N/p1574263863001400